### ISSUE TITLE
Covid FAQ updates — Claim exams

### ISSUE THREAD

**Issue by laurwill** [department-of-veterans-affairs/va.gov-team#47842]
## Issue Description
We need to update the [Covid FAQ page](https://www.va.gov/coronavirus-veteran-frequently-asked-questions/#claims-and-applications) to reflect that in-person claim exams are now available everywhere.

---
## Tasks
- [x] Suggest updates and sections to remove (Laura)
- [x] Review (Danielle)
- [x] Send to stakeholders (Laura)
- [x] Copyedit with prepub (Megan)
- [x] Publish (Megan)

## Acceptance Criteria
- [x] FAQ page is updated

**Comment by laurwill** [comment]
Documenting suggested updates from Heather (claim exams stakeholder): 
- Is VA still doing claim exams? Update to reflect both VA medical facilities and contract examiners are conducting examinations without restrictions. Remove "if we can't conduct the exam in person" content.
- For the 3 Qs in screenshot below, "recommend reaching out to OFO/OPO to gain clarity as to the regional office openings/closures. The offices implemented a “by appointment only system” with an online scheduling system called “[VERA](https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fvets.force.com%2FVAVERA%2Fs%2F&data=05%7C01%7C%7Cd18dafe4d2504117b83808da9d995f13%7Ce95f1b23abaf45ee821db7ab251ab3bf%7C0%7C0%7C637995579289248894%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=vlB3G3rVwJu3AwlfcXOrh%2BJHPJ8Iq853Zaud23%2BwCt0%3D&reserved=0)” and this is still being widely used."
<img width="745" alt="image" src="https://user-images.githubusercontent.com/104158104/193906994-fb16f02e-e226-418e-a395-d28ab022846a.png">

**Comment by laurwill** [comment]
Update: I tweaked the claim exam section based on info from stakeholders Heather and Melanie, and sent for their review. I also removed the COVID alert and link to the FAQ from the claim exams page.

We're going to hold off on adding the VERA tool or making any updates to other sections for now. 

I'm also splitting the facility status updates into a separate ticket, since those won't go live for at least a month.
